packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -452,15 +450,7 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   final EdgeInsets scrollPadding;
 
-  /// {@template flutter.widgets.editableText.enableInteractiveSelection}
-  /// If true, then long-pressing this TextField will select text and show the
-  /// cut/copy/paste menu, and tapping will move the text caret.
-  ///
-  /// True by default.
-  ///
-  /// If false, most of the accessibility support for selecting text, copy
-  /// and paste, and moving the caret will be disabled.
-  /// {@endtemplate}
+  /// {@macro flutter.widgets.editableText.enableInteractiveSelection}
```

Comment:
The `@template` definition of this macro should appear in rendering/editable.dart.

At the moment it does appear to be defined anywhere.